### PR TITLE
Improve documentation of environment variables

### PR DESCRIPTION
**Pull request by eric-wieser**
Let's see if this renders properly...

Brought on by discovering the undocumented coverage feature

### CODE REVIEW

File: documentation/requirements.txt [hunk 1]
```diff
@@ -4,4 +4,5 @@ sphinx-rtd-theme
 cairosvg
 breathe
 sphinxcontrib-trio
+sphinxcontrib-makedomain
```
Comment (eric-wieser) [inline documentation/requirements.txt]: It's possible this will fail due to https://github.com/sphinx-contrib/domaintools/issues/3...

**Comment by imphil** [comment]
Thanks for the code change. The build fails currently, let's wait for https://github.com/sphinx-contrib/domaintools/pull/4 to be merged (or find another solution if that doesn't happen in the next couple of days)

**Comment by eric-wieser** [comment]
I'll split out the last commit so that this can be merged sooner

**Comment by eric-wieser** [comment]
Split to #983, hopefully this PR now works without needing to wait

**Comment by imphil** [comment]
Thanks @eric-wieser! The markup needed a couple additional newlines, I've added those and pushed to your branch. You can test the HTML documentation with `cd documentation; make html; firefox build/html/index.html`

**Comment by imphil** [comment]
Rebased and merged now.

**Comment by eric-wieser** [comment]
RTD doesn't seem to be rebuilding with these changes: https://readthedocs.org/projects/cocotb/builds/

Is it configured to run automatically?

**Comment by imphil** [comment]
> Is it configured to run automatically?

no, it's not (and it can't be fixed currently). See #736

**Comment by imphil** [comment]
I just did a manual rebuild.

**Pull request closed without merging**